ci/scripts/go_bench_adapt.py

```diff
@@ -30,7 +30,17 @@
 
 ARROW_ROOT = Path(__file__).parent.parent.parent.resolve()
 SCRIPTS_PATH = ARROW_ROOT / "ci" / "scripts"
-RUN_REASON = "commit" if os.environ.get("CONBENCH_REF") == "master" else "branch"
+
+if os.environ.get("CONBENCH_REF") == "master":
```

Comment:
is `CONBENCH_REF` set in the workflow?

Reply:
Yes, here:
https://github.com/apache/arrow/blob/794aefd6409a3da339ecc1a3d1ea6466b3d9ddb6/.github/workflows/go.yml#L91